tests/testthat/test-palette_utils.R

```diff
@@ -0,0 +1,35 @@
+# Data Preparation
+library(hubExamples)
+
+## Model output - Scenario
+projection_data <-
+  dplyr::mutate(scenario_outputs,
+                target_date = as.Date(origin_date) + (horizon * 7) - 1)
+projection_data_a_us <-
+  dplyr::filter(projection_data, scenario_id == "A-2021-03-05",
+                location == "US")
+
+## Target data
+target_data_us <-
+  dplyr::filter(scenario_target_ts, location == "US",
+                date < min(projection_data$target_date) + 21,
+                date > "2020-10-01")
+
+
+# Test input information
+test_that("Palette Input parameters", {
+
+  ## Palette/Color input
+  ### One color
+  expect_snapshot(plot_step_ahead_model_output(projection_data_a_us,
+                                               target_data_us,
+                                               show_plot = FALSE,
+                                               pal_color = NULL,
+                                               one_color = "set2"))
```

Comment:
The snapshot tests are great if you want to move fast, but this is not a great indicator of what the test is actually doing. 

Instead, I would suggest to use a part of the actual warning message to test that it's producing the correct warning. 

Wrapping it in `suppressWarnings()` prevents R from complaining about warnings during tests. 

```suggestion
  suppressWarnings({
    expect_warning(plot_step_ahead_model_output(projection_data_a_us,
                                                 target_data_us,
                                                 show_plot = FALSE,
                                                 pal_color = NULL,
                                                 one_color = "set2"),
      "is not one of the accepted color name"
    )
  })
```

Reply:
Done! Thank you for the feedback!!